Documentation/HowTo/adding-disqus-comments-to-posts.md

```diff
@@ -0,0 +1,44 @@
+# How to add Disqus comments to posts
+
+## Getting Short Name from Disqus
+
+Before installing, make sure you've [registered](https://disqus.com/register/) a Disqus [shortname](https://help.disqus.com/customer/portal/articles/286833), and this will be used to reference all of your comments and settings.
+
+
+Once you have got your short name, you can proceed to the next step
+
+## JS
+
+Create a new file called `disqus.js` in your assets folder and paste this code
+
+```javascript
+(function() {  // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        
+        s.src = 'https://REPLACE-WITH-SHORTNAME.disqus.com/embed.js';
+        
+        s.setAttribute('data-timestamp', +new Date());
+        (d.head || d.body).appendChild(s);
+    })();
+```    
+
+Do not forget to replace `REPLACE-WITH-SHORTNAME` with your shortname
+
+## Swift
+
+In you `Theme.Swift` add this to the `makeItemHTML` function, this ensures that the comment thread is only shown on posts and not on Index pages
```

Comment:
`In you` -> `In your`.
`Theme.Swift` -> `theme file` (you don't know what the user calls their theme Swift file).
Also end this sentence with a colon, `:`.